docs/odcs.rst

```diff
@@ -36,6 +36,10 @@ signature of packages will not be checked causing the compose to potentially
 include unsigned packages. If ODCS integration is not in place, this parameter
 has no effect.
 
+``compose_id [ID [ID ...]]`` List of ODCS compose IDs to be used. When defined,
+``compose`` section of ``container.yaml`` is ignored. Cannot be used with
+``repo-url``, nor ``odcs_signed`` parameters.
```

Comment:
``yum_repourls``?

Reply:
I'm going to clean this up, but yes and no :)
`repo-url` is the CLI option, while `yum_repourls` is the API option.

Reply:
Added API/CLI Changes sections to clarify this.